Makefile

```diff
@@ -15,14 +16,20 @@ ci: get_tools get_vendor_deps install test_cover test_lint test
 ########################################
 ### Build/Install
 
-check-ledger: 
+check-ledger:
 ifeq ($(LEDGER_ENABLED),true)
-ifndef GCC
-$(error "gcc not installed for ledger support, please install or set LEDGER_ENABLED to false in the Makefile")
-endif
+   	ifeq ($(UNAME_S),OpenBSD)
```

Comment:
Is this change intended to be in this PR?

Reply:
NO :*(